chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -101,12 +101,14 @@
 
 <!ENTITY zotero.preferences.quickCopy.caption				"Quick Copy">
 <!ENTITY zotero.preferences.quickCopy.defaultOutputFormat	"Default Output Format:">
+<!ENTITY zotero.preferences.quickCopy.locale				"Bibliography Locale:">
```

Comment:
And why not "Language:" ? As in https://www.zotero.org/support/supported_languages and probably a bunch of forum posts.
(and btw *zotero.preferences.quickCopy.locale* and *zotero.bibliography.locale.label* have the same value: shouldn't they be merged ?  – *zotero.preferences.quickCopy.siteEditor.locale* has a colon which distinguish it.)